indykite_sdk/model/is_authorized.py

```
@@ -8,11 +8,14 @@ def deserialize(cls, message):
         if message is None:
             return None
 
-        is_authorized_response = IsAuthorizedResponse(
-            decision_time=timestamp_to_date(message.decision_time),
-            decisions=MessageToDict(message.decisions)
-        )
-
+        message_dict = MessageToDict(message, preserving_proto_field_name=True)
+        if message_dict and message_dict["decisions"]:
+            is_authorized_response = IsAuthorizedResponse(
+                decision_time=timestamp_to_date(message.decision_time),
+                decisions=message_dict["decisions"]
+            )
+        else:
+            is_authorized_response = IsAuthorizedResponse(decision_time=None, decisions={})
```

Are we sure we want to return` IsAuthorizedResponse(decision_time=None, decisions={})`, if I understand correctly we get in the `else` only if the response from the Indykite platform does not match the expected proto message, we should maybe raise an error saying we did not understand the response ?

---

removed